racket/collects/net/http-client.rkt

```diff
@@ -121,34 +129,37 @@
                          #:headers [headers-bs empty]
                          #:content-decode [decodes '(gzip)]
                          #:data [data #f])
+  (define (print-to fmt . args)
+    ;; (eprintf "print-to: ~a~%" (apply format fmt args))
+    (apply fprintf to fmt args))
```

Comment:
I see your point re `apply` and debugging, but *I* don't like seeing `(fprintf to ...)` repeatedly. It smacks of not “do not repeat yourself”. Can I keep `(define printto (curry fprintf to))` ?